packages/core/admin/ee/admin/src/pages/SettingsPage/pages/ReviewWorkflows/EditPage.tsx

```diff
@@ -134,34 +149,40 @@ export function ReviewWorkflowsEditView() {
             return {
               ...stage,
               permissions: hasUpdatedPermissions ? stage.permissions : undefined,
-            };
+            } as Stage;
```

Comment:
I think this is better done with satisfies that way we're checking, not enforcing